vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,295 @@
+
+### Step 2.1: Create a `CMakeLists.txt` file
+
+When using CMake, you can take advantage of the automatic integration through the [vcpkg toolchain
+file](../users/buildsystems/cmake-integration.md#cmake_toolchain_file).
+
+Let's add the following `CMakeLists.txt` file in the project folder:
+
+`CMakeLists.txt`:
+
+:::code language="cmake" source="../examples/snippets/manifest-mode-cmake/CMakeLists.txt":::
+
+### Step 2.2: Configure the project
+
+Add `-DCMAKE_TOOLCHAIN_FILE=<path/to/vcpkg>/scripts/buildsystems/vcpkg.cmake` as a parameter to the
+CMake configure call and vcpkg will be automatically invoked to install the manifest dependencies.
+
+The command should look something like this: `cmake -B <build directory> -S <source directory> -DCMAKE_TOOLCHAIN_FILE=<path/to/vcpkg>/scripts/buildsystems/vcpkg.cmake`
+
+```Console
+PS D:\projects\manifest-example> cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=D:\vcpkg\scripts\buildsystems\vcpkg.cmake
+-- Running vcpkg install
+Detecting compiler hash for triplet x64-windows...
+The following packages will be built and installed:
+    cxxopts:x64-windows -> 3.1.1
+    fmt:x64-windows -> 10.0.0
+    range-v3:x64-windows -> 0.12.0#1
+  * vcpkg-cmake:x64-windows -> 2023-05-04
+  * vcpkg-cmake-config:x64-windows -> 2022-02-06#1
+<output ommited for brevity>
+-- Running vcpkg install - done
+-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19041.
+-- The CXX compiler identification is MSVC 19.27.29111.0
+-- Detecting CXX compiler ABI info
+-- Detecting CXX compiler ABI info - done
+-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
+-- Detecting CXX compile features
+-- Detecting CXX compile features - done
+-- Configuring done
+-- Generating done
+-- Build files have been written to: D:/projects/manifest-example/build
+```
+
+### Step 2.3: Build the project (CMake)
+
+Once the configure step is completed, build the project with `cmake --build <build directory>`:
+
+```Console
+PS D:\projects\manifest-example> cmake --build build
+Microsoft (R) Build Engine version 16.7.0+b89cb5fde for .NET Framework
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+  Checking Build System
+  Building Custom Rule D:\projects\manifest-example\CMakeLists.txt
+  main.cxx
+  fibo.vcxproj -> D:\projects\manifest-example\build\Debug\fibo.exe
+  Building Custom Rule D:\projects\manifest-example\CMakeLists.txt
+```
+
+### [Manual installation](#tab/manual-installation)
+
+> [!div class="nextstepaction"]
+> [Learn more about manually integrating vcpkg in your build system](../users/buildsystems/manual-integration.md)
+
+In a [Developer Command Prompt](/visualstudio/ide/reference/command-prompt-powershell) open to the
+project's folder run the `vcpkg install` command.
+
+```Console
+PS D:\projects\manifest-example> D:\vcpkg\vcpkg.exe install
+Detecting compiler hash for triplet x64-windows...
```

Comment:
AFAIU `vcpkg install` requires that the user to setup target and host triplet environment variables. Are these variables initialized by a "Developer Command Prompt open to the project's folder"?
(Isn't the default triplet still x86-windows at the moment?)
And manual installation may be relevant for user which do not use Visual Studio, i.e. they do not have [Developer Command Prompt](/visualstudio/ide/reference/command-prompt-powershell).

The manual approach can handle only one active target triplet in `vcpkg_installed`. We regularly see users being surprised by packages being removed when changing the target, so it may deserve a mention.